_data/simple-icons.json

```diff
@@ -7383,6 +7383,11 @@
             "source": "https://nordvpn.com/press-area/",
             "guidelines": "https://nordvpn.com/press-area/"
         },
+        {
+            "title": "Norton",
+            "hex": "FFE01A",
+            "source": "https://static.nortoncdn.com/static/ngpcdn/static/ngp2022.3.14.8/dist/images/norton-black.svg"
```

Comment:
From where exactly did you extract this URL? I only see the the logo in the navbar of https://us.norton.com/, and the file URL there is https://us.norton.com/content/dam/norton/logo/logo_norton_d.svg.

According to the [source guidelines of the contribution guide](https://github.com/simple-icons/simple-icons/blob/d4dd8016f0f8cf92a6aa6b92e686b7c9889bc378/CONTRIBUTING.md#source-guidelines):

> If the SVG is sourced from:
> ...
> * **Company website:** If the SVG is found on the company website (but there is no branding page) the source URL should link to a common page, such as the home page or about page, that includes the source image and not the image file itself.
> ...

For this reason, please update the source URL to the website on which includes the source image instead of using the URL of the image file.